external/implementations/samlconf-keycloak-impl/README.md

```diff
@@ -10,9 +10,9 @@
     * Create two new xml files (doesn't matter what they're named)
     * From the `samlconf-sp-metadata.xml` file, copy the first `EntityDescriptor` element and all of its contents. Note: Make sure **not** to include the `EntitiesDescriptor` element or the second `EntityDescriptor` element.
     * Paste that into the first new xml file.
-    * Do the same copy-paste steps but copying the **second** `EntityDescriptor` element into the second new xml file.
+    * Do the same copy-paste steps but copying the **second** `EntityDescriptor` element into the second new xml file. Delete the XML comment inside this `EntityDescriptor` since this will cause issues for Keycloak parsing it.
```

Comment:
❓ Is the comment in the XML necessary in the first place?

Reply:
This is the comment. What do you think?
`<!-- Using the samlhost cert for encryption for now because "DSA" algorithm is not supported by Keycloak for encryption. Could revisit later when DDF support encrypting SAML elements. -->`